.graphqlconfig.yml

```diff
@@ -1,18 +0,0 @@
-projects:
```

Comment:
I know how to get rid of your "providers" error.

![Screenshot from 2022-11-22 01-38-23](https://user-images.githubusercontent.com/1193222/203179713-08bf5266-5976-40bc-9e44-9d320cb6ebde.png)

- create a `fakeschema.graphql` file which needs to be empty
- restore this file
- replace `schemaPath` with the path to the empty fake schema

Error gone and amplify's internal codegen will do nothing since the file is empty. :tada: 

For even more bonus points, in an effort to keep everything clean, create both this file, and the fake schema just inside the doker container, so that the codebase won't get polluted by useless files, and yet amplify's requirements will be met.